spec/controllers/users_controller_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+describe UsersController, :type => :controller do
+  it 'can create user' do
+    expect do
+      post :create, user: {fullname: 'Jo Jo', email: 'jojo@example.com', displayname: 'jojo', password: 'jojojojo', password_confirmation: 'jojojojo' }
+    end.to change(User, :count).by(1)
+    response.should be_redirect
+  end
+
+  it 'cannot create admin' do
+    expect do
+      post :create, user: {fullname: 'Jo Jo', email: 'jojo@example.com', displayname: 'jojo', password: 'jojojojo', password_confirmation: 'jojojojo', role: 'admin' }
```

Comment:
Line is too long. [166/80]<br>Space inside { missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.